dev/doc/newsyntax.tex

```diff
@@ -498,7 +498,7 @@ \subsubsection{Infixes extensibles}
 
 \subsubsection{Gestion des précédences}
 
-Les infixes peuvent être soit laissé à Camlp4, ou bien (comme ici)
+Les infixes peuvent être soit laissé à Camlp5, ou bien (comme ici)
```

Comment:
I apparently missed this change in a file that is kept for historic reasons but shouldn't be changed.
Given how our CI is crowded, I guess if you update this commit to remove just this change, @maximedenes can agree to merge without waiting again for all the tests to pass...

Reply:
Done.